### PR TITLE
Scan Job to use multiple subscription id to run on each subscription id and generate report

### PR DESCRIPTION
**Pull request by MadhviAtroliya**
### :chains: Related Resources
- #5676 

### :+1: Definition of Done
- Add Input text box in settings->node integration->azure->azure-api
- Input text box for subscription_id will not be shown in azure-api
- Create new integration (Azure API)
- Create scan jobs Compliance -> Scan Job  
- Check report subscription_id should be correct as per Azure subscriptions.
Screenshot 2021-09-23 at 7 20 21 PM

### :athletic_shoe: How to Build and Test the Change
- For UI
``` 
STEP 1
inside the hab studio

[default:/src:0]# build components/automate-ui-devproxy/
[default:/src:0]# start_automate_ui_background
[default:/src:0]# start_all_services

STEP 2
open new window
go to automate UI path

$ cd components/automate-ui
and run the command 

npm run serve:hab

```

- Backend
``` rebuild components/compliance-service```
``` rebuild components/nodemanager-service```
Screenshot 2021-09-23 at 7 19 32 PM

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/automate-ui/angular.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
+    "analytics": "14bcc4ea-9146-4d84-96d6-045a5de6702d"
+  },
```
Comment (vivekshankar1) [inline components/automate-ui/angular.json:5]: Is this required ?

Comment (MadhviAtroliya) [reply components/automate-ui/angular.json:5]: Committed by mistake

File: e2e/cypress/integration/ui/settings/data-feed/data-feed-create.spec.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ describe('chef datafeed', () => {
 
 
   before(() => {
-    cy.adminLogin('/settings').then(() => {
+      cy.adminLogin('/settings').then(() => {
```
Comment (vivek-yadav) [inline e2e/cypress/integration/ui/settings/data-feed/data-feed-create.spec.ts:21]: this was fine before.

**Review by shivangi-singh1 — APPROVED** [review]

**Review by vivekshankar1 — APPROVED** [review]
Working properly, thanks

**Pull request merged**